src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_safe_repr.py

```diff
@@ -279,8 +281,54 @@ def _repr_dict(self, obj, level, prefix, suffix,
         yield suffix
 
     def _repr_str(self, obj, level):
-        return self._repr_obj(obj, level,
-                              self.maxstring_inner, self.maxstring_outer)
+        try:
+            if self.raw_value:
+                # For raw value retrieval, ignore all limits.
+                if isinstance(obj, bytes):
+                    yield obj.decode('latin-1')
+                else:
+                    yield self._convert_to_unicode_or_bytes_repr(repr(obj))
```

Comment:
The semantics of "raw value" in VS is that it is *not* a repr - i.e. it shouldn't have any quotes around it, escaping etc.

Reply:
Done (just returning the object directly in this case).